Draft: Tip21 Adding folder and file to updated tips

Roseywinter opened the pull request:
issue #6 

Adding Tip 21 folder and markdown file to updated tips.
No code needed to be verified; content only.
Updated some content and flagged broken links and deprecated resources.
Further content review needed.

- [x] Fix broken links and replace as needed
- [x] Replace deprecated resources with newer resources

aquaomarine commented:
Did we want to change the template to fit the new template? 

If so this tip may need:

- [ ] Intro
- [ ] Getting rid of Outro?

Roseywinter commented:
> Did we want to change the template to fit the new template? 
> 
> If so this tip may need:
> 
> - [x] Intro
> - [x] Getting rid of Outro?

Added Intro. 
Removed Outro. 
Edited Summary.

janackeh reviewed (approved):
Edited and updated links and content
